src/main/java/org/myrobotlab/service/InMoovEyelids.java

```diff
@@ -173,7 +174,7 @@ public long getLastActivityTime() {
   }
 
   public String getScript(String inMoovServiceName) {
-    return String.format("%s.moveEyelids(%d,%d)\n", inMoovServiceName, eyelidleft.getPos(), eyelidright.getPos());
+    return String.format((Locale)null,"%s.moveEyelids(%.2f,%.2f)\n", inMoovServiceName, eyelidleft.getPos(), eyelidright.getPos());

```

Comment:
I think this string format line is wrong.. it should be the string that you're formatting , followed by the arguments to fill that string.  Here you're casting a null value to locale..  this code is not correct.